input/fsh/DEF_Rulesets.fsh

```diff
@@ -40,6 +40,14 @@ RuleSet: Identifiers
   * system 1..1 MS
   * value 1..1 MS
 
+RuleSet: BodySiteQualifierAndLaterality
+* bodySite.extension contains
+     BodyLocationQualifier named locationQualifier 0..*   and
+     LateralityQualifier named lateralityQualifier 0..1
+* bodySite.extension[locationQualifier] ^short = "General location qualifier (excluding laterality) for this bodySite"
+* bodySite.extension[locationQualifier] ^definition = "General location qualifier (excluding laterality) for this bodySite"
+* bodySite.extension[lateralityQualifier] ^short = "Laterality qualifier (excluding laterality) for this bodySite"
```

Comment:
Copy paste errors. Should not say (excluding laterality) on the laterality extension.